src/QsCompiler/CompilationManager/CompilationUnit.cs

```diff
@@ -143,6 +153,8 @@ public References(ImmutableDictionary<NonNullable<string>, Headers> refs, Action
     /// </summary>
     public class CompilationUnit : IReaderWriterLock, IDisposable
     {
+        internal static readonly NameDecorator ReferenceDecorator = new NameDecorator("QsReference");
```

Comment:
Looks like this one can actually be private.

Reply:
It's used by the linking tests.

Reply:
Hu, I could have sworn I built it. But I'll take your word for it.